includes/classes/Rest/IssuesAPI.php

```
@@ -0,0 +1,615 @@
+	}
+
+	/**
+	 * Prepare a single issue output for response.
+	 *
+	 * @param array            $item    Issue object.
+	 * @param \WP_REST_Request $request Request object.
+	 * @return \WP_REST_Response
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		$data = [
+			'id'            => (int) $item->id,
```

PHPStan (2.1.15)

Cannot access property $id on array.


(property.nonObject)